tests/DateTimeLocalTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use AdamWathan\Form\Elements\DateTimeLocal;
+
+class DateTimeLocalTest extends PHPUnit_Framework_TestCase
+{
+    use InputContractTest;
+
+    protected function newTestSubjectInstance($name)
+    {
+        return new DateTimeLocal($name);
+    }
+
+    protected function getTestSubjectType()
+    {
+        return 'datetime-local';
+    }
+}
```

Comment:
Can you add an explicit test here for the DateTime object formatting? Something like `testDateTimeValuesAreBoundAsFormattedStrings` instead of just implicitly using the model binding test.